src/dialog/actorWizardApplication.ts

```diff
@@ -0,0 +1,60 @@
+import { RqgActor } from "../actors/rqgActor";
+import { ActorTypeEnum } from "../data-model/actor-data/rqgActorData";
+import { RQG_CONFIG } from "../system/config";
+import { localize } from "../system/util";
+
+export class ActorWizard extends FormApplication {
+  actor: RqgActor;
+  constructor(object: RqgActor, options: any) {
+    super(object, options);
+    this.actor = object;
+  }
+
+  static get defaultOptions(): FormApplication.Options {
+    return mergeObject(FormApplication.defaultOptions, {
+      classes: ["rqg", "sheet", ActorTypeEnum.Character],
+      popOut: true,
+      template: "systems/rqg/dialog/actorWizardApplication.hbs",
+      id: "actor-wizard-application",
+      title: localize("RQG.ActorCreation.AdventurerCreationWizardTitle"),
+      width: 850,
+      height: 650,
+      tabs: [
+        {
+          navSelector: ".creation-sheet-tabs",
+          contentSelector: ".sheet-body",
+          initial: "0-species",
+        },
+      ],
+      dragDrop: [{ dragSelector: ".item-list .item", dropSelector: null }],
+    });
+  }
+
+  getData(): any {
+    //TODO: this should probably have a specific class
+    return {
+      actor: this.actor,
+    };
+  }
+
+  activateListeners(html: JQuery<HTMLElement>): void {
+    super.activateListeners(html);
+
+    this.form?.querySelectorAll("[data-actor-creation-complete]").forEach((el) => {
+      el.addEventListener("click", (ev) => {
+        this._setActorCreationComplete();
+      });
+    });
+  }
+
+  _setActorCreationComplete() {
+    this.actor.setFlag(RQG_CONFIG.flagScope, RQG_CONFIG.actorWizardFlags.actorWizardComplete, true);
```

Comment:
The "Adventurer Creation" header button is not removed by this. It is no longer visible if I close and reopen the actor though. 
I tried forcing a rerender of the actorsheet by manually doing a actor.render(true) but that didn't remove the window header button either. Not sure how to fix this but it seems the window header is not reinitialised with just a rerender.

Reply:
I really need to get some sleep now, but I found this chat from Forien that relates to this https://discord.com/channels/170995199584108546/527196252576808960/729472401280204800 
Haven't looked into if / how he solved it...

Reply:
Done